### PR TITLE
Add recipe for iperf-2.0.9

### PR DESCRIPTION
**Pull request by theunrealgeek**
Adding a recipe for iperf v2.0.9. 
iperf is a common network bandwidth testing tool especially for TCP. 

Was pretty easy to port to Haiku!

### CODE REVIEW

File: net-analyzer/iperf/iperf-2.0.9.recipe [hunk 1]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="Network bandwidth testing tool"
+DESCRIPTION="iperf2 is the legacy network bandwidth testing tool that support \
+active measurement of bandwidth for protocols such as TCP and UDP and a \
+variety of their parameters"
+HOMEPAGE="https://sourceforge.net/projects/iperf2"
+COPYRIGHT="Board of Trustees of University of Illinois"
+LICENSE="UIUC"
+REVISION="1"
+IPERF_VERSION="2.0.9"
```
Comment (diversys) [inline net-analyzer/iperf/iperf-2.0.9.recipe]: This is the same as $portVersion, so use that

File: net-analyzer/iperf/iperf-2.0.9.recipe [hunk 2]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="Network bandwidth testing tool"
+DESCRIPTION="iperf2 is the legacy network bandwidth testing tool that support \
+active measurement of bandwidth for protocols such as TCP and UDP and a \
+variety of their parameters"
+HOMEPAGE="https://sourceforge.net/projects/iperf2"
+COPYRIGHT="Board of Trustees of University of Illinois"
```
Comment (waddlesplash) [inline net-analyzer/iperf/iperf-2.0.9.recipe]: Should add year here

**Comment by waddlesplash** [comment]
Looks all good besides mine and Diver's comments. And welcome! Always nice to see new contributors.

**Comment by theunrealgeek** [comment]
I have made the changes requested. 
I went with the commit amend+force push option mentioned in the wiki instead of a new commit.

**Pull request merged**